x/evm/keeper/erc20_test.go

```
@@ -11,60 +11,93 @@
+	s.Run("successfully mint 69420 tokens", func() {
+		evmObj, stateDB := deps.NewEVM()
+		_, err := deps.EvmKeeper.ERC20().Mint(
+			erc20,                  /*erc20Addr*/
+			evm.EVM_MODULE_ADDRESS, /*sender*/
+			evm.EVM_MODULE_ADDRESS, /*recipient*/
+			big.NewInt(69_420),     /*amount*/
+			deps.Ctx,
+			evmObj,
+		)
 		s.Require().NoError(err)
+		stateDB.Commit()
```

golangci-lint (1.62.2)

[error] Error return value of `stateDB.Commit` is not checked

(errcheck)